src/libraries/System.Private.CoreLib/src/System/Array.cs

```diff
@@ -1045,7 +1045,7 @@ public static int BinarySearch<T>(T[] array, int index, int length, T value, ICo
         // index into the array.
         //
         // This method is to support the ICollection interface, and calls
-        // Array.Copy internally.  If you aren't using ICollection explicitly,
+        // Array.Copy internally. If you aren't using ICollection explicitly,
```

Comment:
Thanks, but it's very common to use two spaces after a period. It's very unlikely this is the only one in the codebase.

Reply:
Wow, it is the first time ever I heard this. I didn't know that was common thing. Out of that on all codes, I didn't see it.